include/meta/embeddings/word_embeddings.h

```diff
@@ -108,6 +123,12 @@ class word_embeddings
 
     /// The embeddings matrix
     util::aligned_vector<double> embeddings_;
+
+     /// The target vectors matrix
+    util::aligned_vector<double> target_vectors_;
+
+    /// The context vectors matrix
+    util::aligned_vector<double> context_vectors_;
```

Comment:
Nope. I made this refactor without realizing the mode switch was in there. 

Discovered that will debugging and forgot to remove it. Should be gone now.